main.py

```diff
@@ -641,7 +640,9 @@ def list_s3_bucket_matching_prefixes(bucket_name, prefixes):
 
     # sort in reverse date order
     file_keys = sorted(file_keys, key=lambda file: file["sorttime"], reverse=True)
-    return file_keys
+
+    max_files_to_display = 1000
```

Comment:
should this be a module-level constant? I know its a quick fix so maybe it doesn't matter

Reply:
yeah, it probably should be, but because it's meant to be temporary (famous last words) I wanted to keep it as close as to where it's being used as possible.